sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/UserIdUtils.java

```diff
@@ -0,0 +1,72 @@
+package com.microsoft.appcenter.utils;
+
+import android.support.annotation.VisibleForTesting;
+
+import static com.microsoft.appcenter.AppCenter.LOG_TAG;
+
+/**
+ * Utility to store and retrieve values for user identifiers.
+ */
+public class UserIdUtils {
+
+    private static final String CUSTOM_PREFIX = "c";
+
+    private static final String PREFIX_SEPARATOR = ":";
```

Comment:
We are using the same separator for authentication. In CS 3.0 ":" is used for all prefixed values, we can move this to a constant class and use one definition for everything.

Reply:
We have this same string defined in a few places, but they're not related.  I don't see one defined for the common schema separator anywhere.  Since it's only used here so far, I think it's OK to define here, the deepest layer at which it's used.  Am I missing something?

Or, are you saying that other fields which we don't have yet will also use this so we should put it in constants?

Reply:
Let's move both occurrences to constants, I'm doing the patch now.